Makefile

```diff
@@ -33,7 +33,7 @@ COMMIT?=$(shell git rev-parse HEAD 2>/dev/null)
 LD_FLAGS:=-X sigs.k8s.io/kind/pkg/cmd/kind/version.GitCommit=$(COMMIT)
 # the output binary name, overridden when cross compiling
 KIND_BINARY_NAME?=kind
-
+DOCKER?=docker
```

Comment:
We should comment here hinting that you might set this to podman for building

Reply:
done